src/components/Register/detailsCard.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Config from 'config';
+import PropTypes from 'prop-types';
+import Button from 'components/Button';
+
+export default class DetailsCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleBlur = this.handleBlur.bind(this);
+    this.handleFocus = this.handleFocus.bind(this);
+    this.state = {
+      passwordLength: 1,
+      passwordExited: 1,
```

Comment:
i'm not completely sure what passwordExited is. it seems like its being used as a boolean, but has a number value. Also not sure what that number value means either